README.md

```diff
@@ -74,8 +84,8 @@ If you'd like to add a new project to `good-first-issue`, you're more than welco
 - Update `data/projects.json`
   - Add your `<project>` as a property of `projects` with an object that includes a `name`, `description`, and a `q` (representing the GitHub search query).
 
-- Update `README.md`
-  - Add your `<project>` at the bottom of the [Projects](#projects) table above with `name` and `project`
+- Update `README.md` by running `npm run markdown`
```

Comment:
This is awesome. I'm curious: do you think we should add a CI check to see if this is done?